README.md

```diff
@@ -145,7 +145,7 @@ All field tags must be:
 | FORCE_OPTIONS_REVIEW | Opens a dialog advising the user to review the options if they have not done so before applying the Preset.|
 | INCLUDE | Inserts data from one or more separate Presets ahead of the CLI commands of this Preset.  Useful to enforce defaults ahead of your commands. See details below.|
 | OPTION | Commands within `OPTION` tags present the user with a checkbox to apply, or not apply, the enclosed commands.  The default check-box behaviour can be specified.  Each `OPTION` must have a unique name. For more info, [click here](https://github.com/betaflight/firmware-presets#OPTION). |
-| OPTION_GROUP | Text to appear before a group of Options. |
+| OPTION_GROUP | Text to appear before a group of Options. The group can be made mutually exclusive by prepending the (Exclusive) directive to the name of the group |
```

Comment:
Please uppercase all `EXCLUSIVE` tags
```suggestion
| OPTION_GROUP | Text to appear before a group of Options. The group can be made mutually exclusive by prepending the (EXCLUSIVE) directive to the name of the group |
```

Reply:
hmm good questions, is it gonna work with capital letters EXCLUSIVE?
I think it's better to have capital letters, since all directives we have are capital letters.

Reply:
It will work regardless of casing, we call `.lowercase()` on it

Reply:
We have to keep syntax consistent either way. Please update.

Reply:
In the code we can leave it regardless of syntax, just need to ask preset authors to do upper case EXCLUSIVE

Reply:
Updated! Let me know if there's any other changes necessary